host-interaction/file-system/open/open-file.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: open file
```

Comment:
yeah I struggled with the best way to go about adding create/open file rules; adding these as a library rule is a great idea. maybe we can combine to a single library rule named `create or open file`?

I have seen a few rules check for one or more of these APIs and think it would be great to capture all of the possible APIs in one or two rules to help tighten up the checks.

Reply:
> maybe we can combine to a single library rule named `create or open file`?

I like this. Please then replace `CreateFile` etc. with the new library rule or open an issue and I can do it later.